token/program-2022/src/extension/confidential_transfer/processor.rs

```diff
@@ -321,22 +320,22 @@ fn process_deposit(
             return Err(TokenError::MintMismatch.into());
         }
 
-        let mut dest_ct_token_account =
+        let mut dest_confidential_transfer_account =
             dest_token_account.get_extension_mut::<ConfidentialTransferAccount>()?;
-        dest_ct_token_account.approved()?;
+        dest_confidential_transfer_account.approved()?;
```

Comment:
Just fyi: expect a PR from me sweeping through all this to spell out "destination" 👼

Reply:
Ah okay, got it.